examples/example_annotations/browse_annotations.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python
+# coding: utf8
+import csv
+import io
+
+import sys
+
+from flexp.browser.html.to_html import ToHtml
+
+import click
+import logging
+
+from flexp.browser import browser
+from flexp.browser.html.generic import (
+    CsvToHtml, ImagesToHtml,
+    TxtToHtml, FilesToHtml, FlexpInfoToHtml, TxtToHtml)
+from flexp.browser.utils import setup_logging
+from os.path import join, split
+import tornado.ioloop
+import tornado.web
+import os
+import os.path as path
+
+
+log = logging.getLogger(__name__)
+
+setup_logging("debug")
+py_version = sys.version_info[0]
+
+
+# custom function to be called in flexp/browser/browser.py
+def annotation_save(value, ajax_handler):
+    key = ajax_handler.get_argument('id')
+    annotation_file = ajax_handler.get_argument('annotation_file')
+
+    annotation_dict = {}
```

Comment:
There is a [shelve library](https://docs.python.org/3/library/shelve.html) - a simple key value storage. But I guess it doesn't make sense to add it to flexp just because of this.